### ISSUE TITLE
IMixedRealityGestureHandler doesn't work with XRSDK in Hololens 2

### ISSUE THREAD

**Issue by khoadng** [microsoft/MixedRealityToolkit-Unity#9380]
## Describe the bug

Using IMixedRealityGestureHandler with XRSDK pipeline doesn't work when deploy to Hololens 2, it does work in Unity Editor.

## To reproduce

1. Create a Unity project
2. Configure the project according to https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithMRTKAndXRSDK.html (Windows Mixed Reality)
3. Add a cube object
4. Add a TextMeshPro as a child of cube object
5. Attach the below script to the cube.
6. Build the project with target device as Hololens, build type as D3D and architecture as x64
7. Build the generated project with Release | ARM64 configuration
8. Perform any gestures the cube object 
9. Nothing happens

```csharp
using Microsoft.MixedReality.Toolkit.Input;
using System;
using System.Collections;
using System.Collections.Generic;
using TMPro;
using UnityEngine;

public class TestGesture : MonoBehaviour, IMixedRealityGestureHandler<Vector3>
{
    public void OnGestureStarted(InputEventData eventData)
    {
        var debugText = GameObject.Find("Text (TMP)").gameObject;
        var text = debugText.GetComponent<TextMeshPro>();
        text.text = $"OnGestureStarted InputEventData";
    }

    public void OnGestureUpdated(InputEventData<Vector3> eventData)
    {
        var debugText = GameObject.Find("Text (TMP)").gameObject;
        var text = debugText.GetComponent<TextMeshPro>();
        text.text = $"OnGestureUpdated InputEventData<Vector3>";
    }

    public void OnGestureCompleted(InputEventData<Vector3> eventData)
    {
        var debugText = GameObject.Find("Text (TMP)").gameObject;
        var text = debugText.GetComponent<TextMeshPro>();
        text.text = $"OnGestureCompleted InputEventData<Vector3>";
    }

    public void OnGestureUpdated(InputEventData eventData)
    {
        var debugText = GameObject.Find("Text (TMP)").gameObject;
        var text = debugText.GetComponent<TextMeshPro>();
        text.text = $"OnGestureUpdated InputEventData";
    }

    public void OnGestureCompleted(InputEventData eventData)
    {
        var debugText = GameObject.Find("Text (TMP)").gameObject;
        var text = debugText.GetComponent<TextMeshPro>();
        text.text = $"OnGestureCompleted InputEventData";
    }

    public void OnGestureCanceled(InputEventData eventData)
    {
        var debugText = GameObject.Find("Text (TMP)").gameObject;
        var text = debugText.GetComponent<TextMeshPro>();
        text.text = $"OnGestureCanceled InputEventData";
    }
}


```

## Expected behavior

The script should receive and display the required messages.

## Your setup 

- Unity Version 2019.4.12f1
- MRTK Version v2.5.1

## Target platform

- HoloLens 2

**Comment by eusebiu** [comment]
I think you should register the handler into the pipeline:
```
public void OnEnable()
{
    CoreServices.InputSystem.RegisterHandler<IMixedRealityGestureHandler<Vector3>>(this);
}
```

**Comment by keveleigh** [comment]
This is blocked by https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7271, which itself is blocked by some missing information on Unity's side.

**Comment by keveleigh** [comment]
Closing as duplicate of #7271.
